src/Mount.cpp

```diff
@@ -1447,6 +1566,52 @@ void Mount::setSpeed(int which, float speedDegsPerSec) {
     #endif
   }
   #endif
+  
+  #if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE)
+    else if (which == FOCUS_STEPS) {
```

Comment:
This indentation doesn't match the previous, dedent by 2 spaces

Reply:
Done.